lib/create-organization-api.js

```diff
@@ -111,6 +116,56 @@ export default function createOrganizationApi ({ http }) {
       .then((response) => wrapSpaceMembershipCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an Invitation in Organization
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Invitation ID
+   * @return {Promise<OrganizationInvitation>} Promise for a OrganizationInvitation in an organization
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganizations()
```

Comment:
this would have to be the singular `getOrganization(id)` for this example to work, I think :) (this method for getting just 1 was added by the extensibility team's PR )

Reply:
on my way! 🚀

Reply:
solved 🙂